vsts/pipelines/ci.yml

```diff
@@ -5,6 +5,30 @@ variables:
 - group: Oryx
 
 stages:
+  - stage: CreateReleaseTag
```

Comment:
Idea is to create the release tag name and use it across all stages in the pipeline but it seems ADO does not have a in-built way to pass in data between stages , so here I am writing out the release tag name to an artifact file which will be later read and set as environment variable in other stages.